environment.yml

```diff
@@ -21,5 +21,8 @@ dependencies:
 - scipy=0.18.1  
 - sqlalchemy=1.1.4
 - pip:
+    - future==0.16.0
```

Comment:
Hmm.. When I used the future line, I got an error. Maybe we should put it as a dependency instead?